lib/gui/app.js

```diff
@@ -94,13 +94,14 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
   const currentVersion = packageJSON.version;
   const currentReleaseType = release.getReleaseType(currentVersion);
   const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates({
-    ignoreSleepUpdateCheck: currentReleaseType !== release.RELEASE_TYPE.PRODUCTION
+    releaseType: currentReleaseType,
+    lastSleptUpdateNotifier: settings.get('lastSleptUpdateNotifier'),
+    lastSleptUpdateNotifierVersion: settings.get('lastSleptUpdateNotifierVersion') || packageJSON.version
   });
 
   if (_.some([
     !shouldCheckForUpdates,
-    process.env.ETCHER_DISABLE_UPDATES,
-    currentReleaseType === release.RELEASE_TYPE.UNKNOWN
+    process.env.ETCHER_DISABLE_UPDATES
```

Comment:
Note that whether we should update or not based on the release type is now being handled by `UpdateNotifierService.shouldCheckForUpdates()`, so the logic should remain the same.